### PR TITLE
chore: add more docs to usagecontrol

### PR DESCRIPTION
**Pull request by omarsilva1**
I added a more detailed explanation to the usage control documentation page.

### CODE REVIEW

File: docs/pages/documentation/v6/usage-control.md [hunk 1]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
 | 14  | Use Data and Delete it After                   | x       | allows data usage within a specified time interval with the restriction to delete it at a specified time stamp
 | 15  | Modify Data (in Transit)                       | -       |
 | 16  | Modify Data (in Rest)                          | -       |
-| 17  | Local Logging                                  | x       | allows data usage if logged to the Clearing House
-| 18  | Remote Notifications                           | x       | allows data usage with notification message
+| 17  | Local Logging                                  | x       | allows data usage and sends logs to the Clearing House
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md]: What if the connector has no `clearing.house.url` set in its `application.properties`? (maybe change wording here)

File: docs/pages/documentation/v6/usage-control.md [hunk 2]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
...
+## Policy In Depth
+The supported policies are further explained in this section, as well as the meaning of each parameter.
+Every supported policy is defined in the `PolicyPattern` ENUM.
+The `Rule` class from the Information Model is used to define usage policies. The DSC matches rule objects to a
+DSC `PolicyPattern` with the `getPatternByRule` method from the `RuleUtils` class.
+The `usagecontrol\RuleValidator` class matches the patterns found in a rule and throws a `PolicyRestrictionException` if
+a policy restriction is detected. As soon as a `PolicyRestrictionException` is thrown, the access will be restricted.
+
+### Provide Access
+**Description:** This policy simply grants access to the resource. (Can be ignored)
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md]: `(Can be ignored)` Probably shouldn't be there.

File: docs/pages/documentation/v6/usage-control.md [hunk 3]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
...
+
+### N Times Usage
+**Parameters:** a  maximum number of accesses
+
+**Description:** This policy counts the access number of the resource and denies access if the
+access number is greater than the maximum number of accesses.
+It is recommended to disable automatic contract negotiation if you plan on using this policy, so that the
+data consumer does not negotiate a new contract once the maximum number of accesses has been reached.
+To disable automatic contract negotiation change the field to ``policy.negotiation=false`` in
+application.properties.
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md:122]: Do we really want to recommend turning off negotiation completely?

File: docs/pages/documentation/v6/usage-control.md [hunk 4]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
...
+
+**Description:** This policy checks if the issuer connector is equal to a specified connector. The connector id that is
+used for this verification is the one provided in the `config.json` file.
+A similar check is performed when a contract is negotiated. For the negotiation the connector id is also checked to be
+the specified contract consumer.
+
+
+### Security Profile Restricted Usage
+**Parameters:** required connector security profile (BASE_SECURITY_PROFILE, TRUST_SECURITY_PROFILE and
+TRUST_SECURITY_PLUS_PROFILE)
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md]: Isn't it `TRUST_PLUS_SECURITY_PROFILE` ?

File: docs/pages/documentation/v6/usage-control.md [hunk 5]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
...
+
+**Description:** This policy checks if the connector has a specific security profile. This is verified by analysing the
+claims of the message received.
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md]: Claims of the DAT or which claims?

File: docs/pages/documentation/v6/usage-control.md [hunk 6]
```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
...
+- N_TIMES_USAGE
+- USAGE_NOTIFICATION
+
+Policies enforced at data provision currently are:
+- PROVIDE_ACCESS
+- PROHIBIT_ACCESS
+- USAGE_DURING_INTERVAL
+- USAGE_UNTIL_DELETION
+- CONNECTOR_RESTRICTED_USAGE
+- SECURITY_PROFILE_RESTRICTED_USAGE)
```
Comment (ghost) [inline docs/pages/documentation/v6/usage-control.md]: theres an unnecessary `)`

File: docs/pages/documentation/v6/usage-control.md (hunk 3 above)
Comment (omarsilva1) [reply docs/pages/documentation/v6/usage-control.md:122]: @SebastianOpriel what do you think?

Comment (SebastianOpriel) [reply docs/pages/documentation/v6/usage-control.md:122]: If there is no better workaround to avoid bypassing this policy, yes.

Comment (ghost) [reply docs/pages/documentation/v6/usage-control.md:122]: oki, than it's fine for me

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**